packages/example/components/chains/aptosStandard/example.tsx

```
@@ -26,18 +26,343 @@ import {
+      console.log('Sender authenticator:', senderAuthenticator);
+      console.log('Secondary authenticator:', secondarySignerAuthenticator);
+
+      const response = await submitTransaction({
+        transaction: transactionToSubmit,
+        senderAuthenticator: senderAuthenticator,
+        additionalSignersAuthenticators: [secondarySignerAuthenticator],
+      });
+
+      apiFromRef.current?.setJsonValue('submitTransactionResponse', response);
+      if(window && window.open && response.hash) {
+        window.open(`https://explorer.aptoslabs.com/txn/${response.hash}`, '_blank');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)